index.js

```diff
@@ -99,6 +99,14 @@ mOcKiNgCaSe.overrideString = () => {
   return mOcKiNgCaSe;
 };
 
+mOcKiNgCaSe.overrideConsole = (options = {}) => {
```

Comment:
Please add simple jsdoc! See example https://github.com/strdr4605/mockingcase/blob/master/index.js#L84

Reply:
@anmol5varma, jsdoc needs to be added before the function declaration.